packages/core/src/utils/assets/cw20/encode-cw20-asset-send-msg.ts

```diff
@@ -0,0 +1,24 @@
+import { MsgExecuteContract } from 'cosmjs-types/cosmwasm/wasm/v1/tx'
+import { EncodedMsg } from '../../../clients'
+import { jsonToBinary, jsonToUtf8 } from '../../encoding'
+import { CW20Asset } from './cw20-asset'
+
+export function encodeCW20AssetSendMsg<
+  const TMsg extends Record<string, unknown> | string,
+  const TAddress extends string = string,
+>(asset: CW20Asset<TAddress>, sender: string, msg: TMsg) {
+  return {
+    typeUrl: '/cosmwasm.wasm.v1.MsgExecuteContract',
```

Comment:
We should create a task for functionizing the execute contract (and maybe other) message creation as well.